streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -184,11 +184,16 @@ public boolean process() {
             final ProcessorNode currNode = recordInfo.node();
             final TopicPartition partition = recordInfo.partition();
 
-            log.trace("{} Start processing one record [{}]", logPrefix, record);
+            if (log.isTraceEnabled()) {
```

Comment:
You can remove the `if(log.isTraceEnabled()){..}` here and below as this already taken care of via the library. So there is no benefit in adding it here